trustmanager/filestore_test.go

```diff
@@ -50,7 +50,99 @@ func TestAddFile(t *testing.T) {
 	}
 }
 
-func TestRemoveFile(t *testing.T) {
+func TestAddAndGetEncrypted(t *testing.T) {
```

Comment:
Could you add a test that does a bitflip on the on-disk file and make sure that decryption fails due to authentication failure?